src/stories/StringDateInput.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {StoryFn} from '@storybook/react';
+
+import {DateInput as DateInputBase, SpecTypes, StringSpec} from '../lib';
+
+import {InputPreview} from './components';
+
+export default {
+    title: 'String/DateInput',
+    component: DateInputBase,
+};
+
+const baseSpec: StringSpec = {
+    type: SpecTypes.String,
+    viewSpec: {
+        type: 'date_input',
+        layout: 'row',
+        layoutTitle: 'Date Input',
+    },
+};
+
+const excludeOptions = [
+    'maximum',
+    'minimum',
+    'format',
+    'enum',
```

Comment:
<img width="1728" alt="image" src="https://github.com/gravity-ui/dynamic-forms/assets/23384737/8a8736bd-0acd-4d6f-b05c-1e49ade13d41">
let's remove the input props